tools/htdocs/components/15_VRResults.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2018] EMBL-European Bioinformatics Institute
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ensembl.Panel.VRResults = Ensembl.Panel.ContentTools.extend({
+  init: function () {
+    var panel = this;
+
+    this.base();
+
+    // Initialise ZMenus on table links
+    this.el.find('a._zmenu').zMenuLink();
```

Comment:
i am not sure this is needed here, as far as i remember there was no zmenu required in the result table. Probably something to confirm.